Makefile

```
@@ -0,0 +1,37 @@
+PREFIX = /usr/local
+SHAREDIR = $(PREFIX)/share/archlinux
+REPO = contrib
+TAG = $(shell git describe --abbrev=0 --tags)
+
+SCRIPTS = \
+	admin/checkservices \
+	aur/review	\
+	package/co-maintainers \
+	security/security-tracker-check \
+
+
+.PHONY: install
+install:
+	install -dm0755 $(DESTDIR)$(SHAREDIR)
+	for script in $(SCRIPTS); do \
+		install -Dm755 $${script} -t $(DESTDIR)$(SHAREDIR)/contrib/$${script%/*}; \
+	done;
+
+.PHONY: uninstall
+uninstall:
+	for script in $(SCRIPTS); do \
+		rm -rf $(DESTDIR)$(SHAREDIR)/contrib/$${script%/*}; \
+	done;
+	rmdir $(DESTDIR)$(SHAREDIR)
+
+.PHONY: tag
+tag:
+	git tag $(shell date +%Y%m%d) || true
```

```suggestion
	git tag -s $(shell date +%Y%m%d) || true
```

PGP signatures are good, let's use signed/annotated tags.

---

Why `|| true`? e.g. what if my gpg key isn't present and the tag fails creation?

---

But I don't want to force empty messages. I want actual messages. :p

As for `|| true`, the git push --tags won't do anything useful if the tag creation failed due to missing gpg keys. But trying to re-upload after the tag is already created, will cause errors from "fatal: the tag already exists" to be ignored.

---

`|| true` because I think it's annoying if `make tag` fails when you run it twice in the same day. Say you setup the remotes incorrectly etc.

`-m ''`  is also a good point.

---

In which case, maybe check if the tag exists already.

---

```suggestion
	git describe --exact-match >/dev/null 2>&1 || git tag -s $(shell date +%Y%m%d)
```